.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+### Issue
+
+_Please describe the issue._
+
+_Optional: add details about technical approach, solutions etc._
+
+
+### Dependencies
+
+_Optional: reference other pull requests etc._
+
+
+### Testing
+
+_Describe how to test._
+
+_Optional: attachments like images, videos, etc._
+
+---
+
+### Checklist
+
+- [ ] Title conforms to [semantic commits messages](https://github.com/wireapp/.github#usage).
+- [ ] Title contains a reference JIRA issue number like `[WPB-XXX]`.
+- [ ] Description is filled and free of optional paragraphs.
+- [ ] Adds/updates automated tests.
+
+---
+
+<!--do not remove the jira markers to link tickets automatically -->
+<!--jira-description-action-hidden-marker-start-->
+
+<!--jira-description-action-hidden-marker-end-->
```

Comment:
should not that be on top?

Reply:
I would rather like to have it on bottom to be able to read the important things first, but not sure if it works. This was the same in the suggested template I asked for feedback some weeks ago...